[2025-08-13T12:32:32.416+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-13T12:32:32.831+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: xrs_update_12h.backfill_ncei_j2 scheduled__2025-08-13T00:00:00+00:00 [queued]>
[2025-08-13T12:32:32.843+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: xrs_update_12h.backfill_ncei_j2 scheduled__2025-08-13T00:00:00+00:00 [queued]>
[2025-08-13T12:32:32.844+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-08-13T12:32:32.873+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): backfill_ncei_j2> on 2025-08-13 00:00:00+00:00
[2025-08-13T12:32:32.888+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=6297) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-08-13T12:32:32.885+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'xrs_update_12h', 'backfill_ncei_j2', 'scheduled__2025-08-13T00:00:00+00:00', '--job-id', '167', '--raw', '--subdir', 'DAGS_FOLDER/xrs_update_daily.py', '--cfg-path', '/tmp/tmp71sso53u']
[2025-08-13T12:32:32.889+0000] {standard_task_runner.py:63} INFO - Started process 6299 to run task
[2025-08-13T12:32:32.889+0000] {standard_task_runner.py:91} INFO - Job 167: Subtask backfill_ncei_j2
[2025-08-13T12:32:32.950+0000] {task_command.py:426} INFO - Running <TaskInstance: xrs_update_12h.backfill_ncei_j2 scheduled__2025-08-13T00:00:00+00:00 [running]> on host def4e694b617
[2025-08-13T12:32:33.132+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='you' AIRFLOW_CTX_DAG_ID='xrs_update_12h' AIRFLOW_CTX_TASK_ID='backfill_ncei_j2' AIRFLOW_CTX_EXECUTION_DATE='2025-08-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-08-13T00:00:00+00:00'
[2025-08-13T12:32:33.133+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-13T12:32:33.158+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-13T12:32:33.162+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/project/src/data/xrs_update.py --mode backfill --start 2025-08-11 --end 2025-08-11']
[2025-08-13T12:32:33.182+0000] {subprocess.py:86} INFO - Output:
[2025-08-13T12:32:35.912+0000] {subprocess.py:93} INFO - [WARN] Aucun fichier pour 2025-08-11 (NCEI listing vide et fallback SunPy sans rÃ©sultat)
[2025-08-13T12:32:35.913+0000] {subprocess.py:93} INFO - [DONE] backfill complete -> rows=0
[2025-08-13T12:32:36.057+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-08-13T12:32:36.060+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-13T12:32:36.109+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=xrs_update_12h, task_id=backfill_ncei_j2, run_id=scheduled__2025-08-13T00:00:00+00:00, execution_date=20250813T000000, start_date=20250813T123232, end_date=20250813T123236
[2025-08-13T12:32:36.163+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-13T12:32:36.200+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-13T12:32:36.203+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
